### ISSUE TITLE
Patch StartBrowser

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#41]
```
Hello,

here's a patch for the start browser utility:

- Enables the JAVA 1.6 desktop browser launch method (StartBrowserDesktop
class), which uses the user's preferences on every platform it is implemented. 
- Adds a property (h2.browserUserAgent) to launch a specific browser.
  Use case: your company mandates an obsolete browser (say IE6), while your
preference goes to a more modern and safer browser (say Firefox).

It keeps the same entry point, StartBrowser.openUrl(url) and fallbacks
nicely with previous Java versions.

I didn't really test the SwitchSource option but it should work.

Please tell me what you think.

Cheers,
Olivier

```

Original issue reported on code.google.com by `opar...@gmail.com` on 23 Oct 2008 at 1:22

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Thanks a lot for your patch! I will apply it, however I will change a few 
things:

- I will move the system property to SysProperties (like the other settings)

- I rename it to h2.browser

- For java.awt.Desktop I will use reflection (as in org.h2.util.Console) 
instead of a
source code switch. That way even the default h2.jar (compiled with JDK 1.4) 
uses it
when running with JDK 1.6.

- I will not use another class

```

Original comment by `thomas.t...@gmail.com` on 25 Oct 2008 at 2:24
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Committed in revision 1010, see also 
http://code.google.com/p/h2database/source/detail?r=1010
```

Original comment by `thomas.t...@gmail.com` on 25 Oct 2008 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
Implemented in version 1.1.103
```

Original comment by `thomas.t...@gmail.com` on 7 Nov 2008 at 3:27
- Changed state: **Fixed**
